lib/dependencies/UnsupportedDependency.js

```diff
@@ -2,20 +2,22 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var NullDependency = require("./NullDependency");
+"use strict";
+const NullDependency = require("./NullDependency");
+const webpackMissingModule = require("./WebpackMissingModule").module;
 
-function UnsupportedDependency(request, range) {
-	NullDependency.call(this);
-	this.request = request;
-	this.range = range;
+class UnsupportedDependency extends NullDependency {
+	constructor(request, range) {
+		super();
+		this.request = request;
+		this.range = range;
+	}
 }
-module.exports = UnsupportedDependency;
-
-UnsupportedDependency.prototype = Object.create(NullDependency.prototype);
-UnsupportedDependency.prototype.constructor = UnsupportedDependency;
 
-UnsupportedDependency.Template = function UnsupportedDependencyTemplate() {};
+UnsupportedDependency.Template = class UnsupportedDependencyTemplate {
+	apply(dep, source, outputOptions, requestShortener) {
+		source.replace(dep.range[0], dep.range[1], webpackMissingModule(dep.request));
+	}
+}
 
-UnsupportedDependency.Template.prototype.apply = function(dep, source, outputOptions, requestShortener) {
-	source.replace(dep.range[0], dep.range[1], require("./WebpackMissingModule").module(dep.request));
```

Comment:
yeah i wasnt sure why it was called inline, after all its "just" a reference so it doesnt really hurt t be imported?